labs/shipped-application-level-policy/4.md

```diff
@@ -0,0 +1,26 @@
+## aie-burnit Usage
+
```

Comment:
Here we should add snapshot of deploy page with link to open aie-burnit app. 
It not very cleat how memory setting page is opened.